libraries/USBHost/examples/USB_desc/USB_desc.ino

```diff
@@ -159,7 +159,7 @@ byte getdevdescr( byte addr, byte &num_conf )
   return( 0 );
 }
 
-void printhubdescr(uint8_t *descrptr, uint8_t addr)
+void printhubdescr(uint8_t *descrptr, __attribute__((unused)) uint8_t addr)
```

Comment:
attribute here as well. Sorry, I should spot it earilier

Reply:
OK.